pkg/api/store/auth/store.go

```diff
@@ -0,0 +1,48 @@
+package auth
+
+import (
+	"fmt"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/norman/types/convert"
+	"github.com/rancher/rancher/pkg/auth/providers/common"
+	corev1 "github.com/rancher/types/apis/core/v1"
+	"github.com/rancher/types/client/management/v3"
+	"strings"
+)
+
+var typeToField = map[string]string{
+	client.GithubConfigType:          "clientSecret",
```

Comment:
do we already have these constants defined somewhere? If yes, we should move them to the common place

Reply:
I don't think we do, we have for types but not with fields